modules/002-deckhouse/openapi/config-values.yaml

```diff
@@ -287,87 +293,107 @@ properties:
 
   registry:
     type: object
-    default:
-      mode: Unmanaged
+    x-doc-search: "registry, docker, container, images, credentials, private registry"
+    description: |
+      Settings for accessing the container registry with Deckhouse images.
 
+      This parameter allows you to configure how Deckhouse connects to the container registry to pull its own component images.
+      You can either rely on pre-configured access on cluster nodes (`Unmanaged` mode) or provide Deckhouse with direct access credentials (`Direct` mode).
     x-examples:
       - mode: Unmanaged
       - mode: Direct
         direct:
           imagesRepo: registry.deckhouse.io/deckhouse/ee
-          license: "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-
+          license: "DECKHOUSE_LICENSE_KEY"
     properties:
       mode:
         type: string
         description: |
-          Registry mode
+          The mode for accessing the container registry with Deckhouse images.
+
+          - `Unmanaged` — the default mode. In this mode, the `registry` module does not run an internal registry. Access to the registry is configured manually using the `change-registry` helper script. **Note:** Changing the registry in this mode will cause a full restart of all Deckhouse components.
+          - `Direct` — a mode for direct access to the registry via a non-caching in-cluster proxy registry. This allows for registry changes without a full component restart.
```

Comment:
remove the non-caching proxy suggestion, we're using a "virtual" registry address that is overwritten on the fly into the real registry address.

Reply:
Also please sync with ru language